test/topology_custom/test_mv_backlog.py

```diff
@@ -76,3 +81,115 @@ async def test_gossip_same_backlog(manager: ManagerClient) -> None:
     await cql.run_async(stmt, [0, 0, 'a'], host=hosts[0])
 
     await cql.run_async(f"DROP KEYSPACE ks")
+
+# A test for the view_flow_control_delay_limit_in_ms parameter.
+#
+# The test creates a table with a materialized view and pins all tablets
+# of the base on one node, and all tablets of the view to another node.
+#
+# Then, for a bunch of some possible values of the parameter, do the following:
+#
+# - Perform a live update of the parameter on both nodes.
+# - Do a large write to the base table while using the `never_finish_remote_view_updates`
+#   error injection to keep the remote view update resident and occupying
+#   memory, influencing the view update backlog as a result.
+#   Follow it with one more small write to make sure that it is delayed
+#   by the first write's backlog.
+# - Read the X metric in order to measure the calculated delay of last view update.
```

Comment:
Did you plan to go back and modify this X in the comment to the real metric name? Or did you write it this way because you check several metrics?

Reply:
I don't remember, but it's quite probable. In any case, changed `X` to `scylla_storage_proxy_coordinator_mv_flow_control_delay_total`.